src/styles/globals.css

```
@@ -27,7 +35,7 @@
 }
 
 body {
-  font-family: 'Geist Sans', sans-serif !important;
```

```suggestion
font-family: 'Geist Sans', sans-serif !important;
```
chakra-ui overwrites font-family body style. That's why its set to `!important`